backfill_corrections/delphiBackfillCorrection/R/main.R

```diff
@@ -56,13 +56,16 @@ run_backfill <- function(df, params,
         coef_list[[key]] <- list()
       }
     }
-
+    
     msg_ts("Splitting data into geo groups")
     group_dfs <- group_split(df, .data$geo_value)
 
     # Build model for each location
     for (subdf in group_dfs) {
       geo <- subdf$geo_value[1]
+      
+      if (!(geo %in% c("pa", "ma", "ny"))) {next} 
```

Comment:
Test code?

Reply:
Oh, yes! Sorry, I forgot to delete it. Ignore it currently.